.github/workflows/unittest.yml

```diff
@@ -24,15 +24,24 @@ jobs:
       - run: go build ./...
       - name: Run tests
         if: matrix.os != 'ubuntu-latest'
-        run: go test ./...
+        run: go test -race ./...
       - name: Run tests
         if: matrix.os == 'ubuntu-latest'
-        run: go test -coverprofile=coverage.out ./...
+        run: go test -race -coverprofile=coverage.out ./...
+      - name: Run benchmark
+        if: matrix.os == 'ubuntu-latest'
+        run: make benchmark-ci
+      - name: Delta
+        uses: hermanschaaf/delta-action@v1.0.7
```

Comment:
We can change this to the original upstream version if/when my PR gets merged: https://github.com/netlify/delta-action/pull/331